jest.config.ts

```diff
@@ -5,7 +5,8 @@ const config: Config.InitialOptions = {
   testMatch: ['**/?(*.)+(test).+(ts|js)'],
   transform: {
     '^.+\\.ts$': 'ts-jest'
-  }
+  },
+  testEnvironment: 'node'
```

Comment:
`node` is the default env, but doesn't hurt to be explicit.
https://jestjs.io/docs/configuration#testenvironment-string